### PR TITLE
feat: added readme file

### PR DESCRIPTION
**Pull request by annton-shakur**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Online Book Store
+The application was designed to automize the processes of the online book store, which include the following:
+* managing the inventory of available books and categories (adding new items, removing outdated ones)
+* populating shopping carts (with cart items) and completing orders based on their contents
+* managing the user's orders (from admins prospective)
+## Technologies used
```
Comment (d1sam) [inline readme.md:6]: it's better to split technologies to categories.

File: readme.md
Comment (d1sam) [inline readme.md:1]: what about sections:
- Elaborate on how someone can set up or use your project. This is a very important part.
- Don't forget to mention any challenges you faced and how you overcame them.
- Possible improvements.

**Review by d1sam — CHANGES_REQUESTED** [review]
Good job, but there are some improvements should be done!

File: readme.md [hunk 2]
```diff
@@ -105,4 +110,18 @@ The application was designed to automize the processes of the online book store,
...
 * **dto**: Data Transfer Objects used for communication between the client and server
-* **mapper**: Mapper interfaces used for conversions between DTOs and entity models
\ No newline at end of file
+* **mapper**: Mapper interfaces used for conversions between DTOs and entity models
+## How to test the project from your side? 
+* Make sure to have JDK 17+, Docker and Postman installed
+* Clone this repository
+* Create the .env file with the corresponding variables
+* Build images using _docker-compose build_ and run the service in containers using _docker-compose up_
+* You are welcome to try out this application options using the next Postman collection: there will be a collection here tomorrow (I hope). Just let me know if the other steps are valid
```
Comment (yevheniisynytsia) [inline readme.md]: please do not forget to add it
it is important for local testing

**Review by yevheniisynytsia — APPROVED** [review]
I like your readme - great job!

**Pull request merged**